tests/test_imageCollection_func.py

```diff
@@ -68,3 +68,9 @@ def test_image_collection_group_by_query():
     response = SQL2GEE(JsonSql(sql).to_json(), geojson=r).response()
     assert len(response) == 10, "BASIC limit query incorrect"
     return
+
+def test_image_collection_group_by_with_geometry_where_condition():
+    sql = "select system:index, first(q25) as q25,first(q50) as q50, first(q75) as q75, mean(q75) as q75_mean from 'projects/resource-watch-gee/cli_051_nexgddp_annual_tasmin' where ST_INTERSECTS(ST_SetSRID(ST_GeomFromGeoJSON('{\"type\":\"Point\",\"coordinates\":[-110.22939224192194,19.986126139624318]}'),4326),the_geom) group by system:index"
```

Comment:
I'd prefer to have a test that runs on top of a public dataset. Running tests on a "private" dataset basically ties the tests to a single GCP account.

Reply:
Not really if it is public read but I can change the dataset to one in Google public dataset if you will feel more confortable ❤️

Reply:
yes, please. So one of the issues we had recently is that we were using our own datasets to test, and that disappeared for whatever reason, so I had to ask you for new ones. "public" datasets do come with their own risks, of changes we dont control, but I still feel like that's a lesser risk